src/scene/mesh-instance.js

```diff
@@ -260,6 +261,17 @@ class MeshInstance {
      */
     castShadow = false;
 
+    /**
+     * Mask describing which enable shadow casting in the cascade {@link ShadowRendererDirectional} for this mesh instance.
```

Comment:
- I would simplify this a bit, no need for too much detail. Maybe see `MeshInstance.visible` 
- do not link to ShadowRendererDirectional, this is a private implementation user knows nothing about. Mention `LIGHTTYPE_DIRECTIONAL` instead
- mention the default value (Defaults to ..), see other properties in the engine